health-services/household/src/main/resources/db/migration/main/V20230310124100__household_member_search_index_ddl.sql

```diff
@@ -0,0 +1,11 @@
+CREATE INDEX IF NOT EXISTS
+    idx_id_householdId_householdClientReferenceId_individualId_individualClientReferenceId_isHeadOfHousehold_tenantId
```

Comment:
If at a later stage, the requirement is to delete an index on a particular field, will it be possible? Or Will we have to delete the whole index?
If it is latter, then it is a problem. In that case, do create indexes for each field.

Reply:
yeah in that case we have to delete the whole index.
so it will be better if we create indexes separately for each field.
Will make that changes.